src/Microsoft.AspNet.Security/DataHandler/Serializer/TicketSerializer.cs

```diff
@@ -38,18 +38,23 @@ public virtual AuthenticationTicket Deserialize(byte[] data)
         public static void Write([NotNull] BinaryWriter writer, [NotNull] AuthenticationTicket model)
         {
             writer.Write(FormatVersion);
-            ClaimsIdentity identity = model.Identity;
-            writer.Write(identity.AuthenticationType);
-            WriteWithDefault(writer, identity.NameClaimType, DefaultValues.NameClaimType);
-            WriteWithDefault(writer, identity.RoleClaimType, DefaultValues.RoleClaimType);
-            writer.Write(identity.Claims.Count());
-            foreach (var claim in identity.Claims)
+            writer.Write(model.AuthenticationScheme);
+            var principal = model.Principal;
```

Comment:
Don't think you need to write the scheme out as information. The use of data protection purposes prevents serialized information going out on one scheme coming back with another.